src/NUnitFramework/framework/Interfaces/IFixtureBuilder.cs

```diff
@@ -40,7 +43,20 @@ public interface IFixtureBuilder
         /// Builds any number of test fixtures from the specified type.
         /// </summary>
         /// <param name="type">The type to be used as a fixture.</param>
-        // TODO: This should really return a TestFixture, but that requires changes to the Test hierarchy.
         IEnumerable<TestSuite> BuildFrom(Type type);
     }
+
+    /// <summary>
+    /// The IFixtureBuilder2 interface extends IFixtureBuilder by allowing
+    /// use of a PreFilter, which is used to select methods as test cases.
+    /// </summary>
+    public interface IFixtureBuilder2 : IFixtureBuilder
```

Comment:
What do you think of `IFilterableFixtureBuilder`?

Reply:
I read something (who knows where) and have avoided suffixing `2` or `Ex` (which winapi does) ever since. Perhaps `IFixtureBuilder2` says, "this would be `IFixtureBuilder` if there was any way to do that. Perhaps `IFilterableFixtureBuilder` says, "this is why you would use this kind of `IFixtureBuilder`."
Not sure which of the two we things we want to be saying.

Reply:
@nunit/framework-team ping?